pages/collections/[id].tsx

```diff
@@ -73,6 +74,8 @@ const Home: NextPage<Props> = ({ fallback, id }) => {
       ? collectionResponse.data[0]
       : undefined
 
+  const collectionActivity = useCollectionActivity(id, [])
```

Comment:
I feel like it makes more sense to put this inside the CollectionActivityTable, so that when it's remounted it will reload the data.